pkg/cmd/cli/install/install.go

```diff
@@ -130,6 +133,82 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+// Return version in the format: vX.Y.Z
+func GetVersionFromImage(containers []v1.Container, imageName string) string {
+	var tag = ""
+	for _, container := range containers {
+		if strings.Contains(container.Image, imageName) {
+			tag = strings.Split(container.Image, ":")[1]
+			break
+		}
+	}
+	if tag == "" {
+		fmt.Printf("Failed to get tag from image %s\n", imageName)
+		return ""
+	}
+	if strings.Contains(tag, "-") {
+		version := strings.Split(tag, "-")[0]
+		return version
+	} else {
+		return tag
+	}
+}
+
+// If currentVersion == minVersion, return 0
+// If currentVersion > minVersion, return 1
+// If currentVersion < minVersion, return 2
+// Assume input string format is vX.Y.Z
+func CompareVersion(currentVersion string, minVersion string) int {
```

Comment:
This is the same as the func in the other PR, right?  This should return -1.  We'll merge the velero version check first.  After we merge the velero version check, rebase this PR

Reply:
Yes there are two func the same as the func in velero version check pr. I'll rebase and resolve conflict after velero version check pr is merged.